app/src/main/java/com/github/wnder/picture/Picture.java

```diff
@@ -1,158 +1,14 @@
 package com.github.wnder.picture;
 
-import android.location.Location;
-
-import com.github.wnder.Storage;
-import com.google.android.gms.tasks.Task;
-import com.google.firebase.firestore.DocumentSnapshot;
-import com.google.firebase.firestore.GeoPoint;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.TreeMap;
-import java.util.function.Consumer;
-
 /**
  * abstract class for all kind of pictures
  */
-public abstract class Picture {
+public final class Picture {
 
     public final static String UNINITIALIZED_ID = "";
```

Comment:
You should find another place to put this string, maybe we could just use a class with all the "public static final" strings that we use often/in a lot of places though I don't know if it's good practice.

Reply:
I don't know either but for now it should be at least placed in a class with a better name, so I will do that.